packages/nextjs/contracts/predeployedContracts.ts

```diff
@@ -2,951 +2,47 @@
-          items: [
-            {
-              name: "totalSupply",
-              type: "function",
-              inputs: [],
-              outputs: [
-                {
-                  type: "core::integer::u256",
-                },
-              ],
-              state_mutability: "view",
-            },
-            {
-              name: "balanceOf",
-              type: "function",
-              inputs: [
-                {
-                  name: "account",
-                  type: "core::starknet::contract_address::ContractAddress",
-                },
-              ],
-              outputs: [
-                {
-                  type: "core::integer::u256",
-                },
-              ],
-              state_mutability: "view",
-            },
-            {
-              name: "transferFrom",
-              type: "function",
-              inputs: [
-                {
-                  name: "sender",
-                  type: "core::starknet::contract_address::ContractAddress",
-                },
-                {
-                  name: "recipient",
-                  type: "core::starknet::contract_address::ContractAddress",
-                },
-                {
-                  name: "amount",
-                  type: "core::integer::u256",
-                },
-              ],
-              outputs: [
-                {
-                  type: "core::bool",
-                },
-              ],
-              state_mutability: "external",
-            },
-          ],
-        },
-        {
-          kind: "struct",
-          name: "openzeppelin::token::erc20_v070::erc20::ERC20::Transfer",
-          type: "event",
-          members: [
-            {
-              kind: "data",
-              name: "from",
-              type: "core::starknet::contract_address::ContractAddress",
-            },
-            {
-              kind: "data",
-              name: "to",
-              type: "core::starknet::contract_address::ContractAddress",
-            },
-            {
-              kind: "data",
-              name: "value",
-              type: "core::integer::u256",
-            },
-          ],
-        },
-        {
-          kind: "enum",
-          name: "openzeppelin::token::erc20_v070::erc20::ERC20::Event",
-          type: "event",
-          variants: [
-            {
-              kind: "nested",
-              name: "Transfer",
-              type: "openzeppelin::token::erc20_v070::erc20::ERC20::Transfer",
-            },
-          ],
-        },
-      ],
+      abi: universalAbi,
       classHash:
         "0x04ad3c1dc8413453db314497945b6903e1c766495a1e60492d44da9c2a986e4b",
     },
   },
+  mainnet:{
+    Eth: {
+      address: universalEthAddress,
+      abi: universalAbi,
+      classHash: universalClassHash,
+    },
+    Strk: {
+      address: universalStrkAddress,
+      abi: universalAbi,
+      classHash: universalClassHash,
+    },
+  }
```

Comment:
add eth & strk for mainnet, Are these values ​​correct? (keep it same as devnet)